src/main/java/com/questhelper/panel/QuestHelperPanel.java

```diff
@@ -78,6 +81,7 @@ public class QuestHelperPanel extends PluginPanel
 
 	private final JPanel allDropdownSections = new JPanel();
 	private final JComboBox<Enum> filterDropdown, difficultyDropdown, orderDropdown;
+	private final JComboBox<Enum> skillSelectionPanel;
```

Comment:
nit: Rename to be consistent with the other dropdowns

```suggestion
	private final JComboBox<Enum> skillFilterDropdown;
```